requirements/edx/local.txt

```diff
@@ -1,12 +1,13 @@
 # Python libraries to install that are local to the edx-platform repo
 -e .
--e common/lib/calc
+
 -e common/lib/capa
--e common/lib/chem
--e common/lib/dogstats
--e common/lib/safe_lxml
--e common/lib/sandbox-packages
--e common/lib/symmath
 -e common/lib/xmodule
 
+-e openedx/core/lib/calc
```

Comment:
Thinking this through some more, it is possible that there are installed packages such as XBlocks that depend upon these libraries. Maybe we should have a different location for packages that are independently installable but not yet moved out into their own repo.

Given all this, I think I'll just close this PR and leave them where they are for now.